### DISCUSSION TITLE
Achievements

### DISCUSSION THREAD

**Discussion by adamhannigan** [sunflower-land/sunflower-land#1320] [Art]
Our players have been grinding hard. We want to introduce a range of achievement badges so the players can show off how hard they have been working.

These can be focussed on any area of the game - farming, chopping, mining, livestock etc.

The aim is to give players extra goals to strive for and the ability to show off their hard work to the community.

**Where will the badges show?**

Each badge will be tokenised and can be displayed on a user's inventory. We will also allow badges to show on a Bumpkin Profile Picture 🚀 

**Contributing**

Are you artist? We would love to see some pixel art submissions in this discussion (see requirements below) 🤗 

Are you a player? Please comment what type of achievements you would like to unlock and show off 🚀 

**Requirements**

We will want an 'icon' and the icon with a badge in separate files. We will likely use a consistent background for the badges (TBD)

- 12x12px base icon. Max 16x16px including the background
- The achievement to unlock (we may need to alter this to meet the game's designs)
- Work is original and not plagiarised
- By posting you agree to the [terms of service](https://docs.sunflower-land.com/support/terms-of-service)

**Deadline**

This will be a long running discussion where players and artists can contribute ideas 🏆

**Comment by TaRaSiSt** [comment]
**BG**
![Bg](https://user-images.githubusercontent.com/47809767/183783005-f157fec2-18a7-42ef-9ec5-8619befa528b.png)

**Some icons can be used for new skills)**

### **Аchievements that do not have a level of rarity:** 

![Movie](https://user-images.githubusercontent.com/47809767/183968142-b28d7e66-7276-4f35-b74c-5eb310b05cbf.gif)

![Start the game](https://user-images.githubusercontent.com/47809767/183783099-f04ed41c-66ac-4424-8d20-6ca51b5d7da5.png) ![Start the gameWB](https://user-images.githubusercontent.com/47809767/183836599-b0d8039f-906a-40f3-848b-84b3ee184607.png) - **Start the game** (Thank you for starting to play Sunflower land. Welcome, enjoy!)
![make friends with goblins](https://user-images.githubusercontent.com/47809767/183783222-793911b8-de08-44c3-a787-96e7d07e10bc.png) ![make friends with goblinsWB](https://user-images.githubusercontent.com/47809767/183836651-f4429711-254d-4021-aeee-503040c3020d.png)  - **Make friends with goblins** (You met goblins! Now they are your friends or not... Who knows these goblins.)
![Find your shovel](https://user-images.githubusercontent.com/47809767/183783326-d1297105-8557-4c17-8e91-8b86b55c4dd3.png) ![Find your shovelWB](https://user-images.githubusercontent.com/47809767/183836724-827e0672-8b85-4df4-acfe-cfbb956966ce.png) - **Find your shovel** (You found your shovel. Now you can plant and harvest!)
![Collector](https://user-images.githubusercontent.com/47809767/183784617-7831dd7a-a316-4253-a8d0-bc5fe61fc5da.png) ![CollectorWB](https://user-images.githubusercontent.com/47809767/183836766-41f85f98-0435-4ab7-8afc-2bdcea717c10.png) - **Collector** (You found/crafted all *something*)
![Diskette](https://user-images.githubusercontent.com/47809767/183785319-21b16e64-65c8-4e7e-b165-99343c7ad175.png) ![DisketteWB](https://user-images.githubusercontent.com/47809767/183836810-3a4ae5d5-83e2-4e38-b2d0-45905a94afd9.png) - **Sync on chain** (Congratulations! You know how to save your progress.)
![Good neighbor](https://user-images.githubusercontent.com/47809767/183787296-c6deda38-2b58-4c34-a730-f308a6f610a1.png) ![Good neighborWB](https://user-images.githubusercontent.com/47809767/183836839-5b045a68-ca09-4906-b1f3-c3e7a08faa2c.png) - **Good neighbor** (Visit someone's farm)
![Get your first skill](https://user-images.githubusercontent.com/47809767/183845380-2c68343a-ca17-41eb-b64d-84d87fe111dd.png) ![Get your first skillWB](https://user-images.githubusercontent.com/47809767/183845399-823517ad-2ee4-4ead-a2e2-09af9eb7fcf2.png) - **Get your first skill** (Congratulations, you have reached the level to choose a skill. Why are you waiting - do it!)
![Style icon](https://user-images.githubusercontent.com/47809767/183848517-8db740b0-b157-41c9-bc3e-9ee86c698558.png) ![Style iconWB](https://user-images.githubusercontent.com/47809767/183848533-757b512b-0d61-4f5e-afca-a60f4106fa32.png) - **Style icon** (Customize your character)
![My Precious](https://user-images.githubusercontent.com/47809767/183959895-1b5571af-1575-4d68-84f2-20925d949cdc.png) ![My PreciousWB](https://user-images.githubusercontent.com/47809767/183960125-87a0550f-34a7-4162-ad3d-2f109257c921.png) - **My Precious** (Why are you taking them away?)
![Cake](https://user-images.githubusercontent.com/47809767/184556348-6f857876-d68a-4720-a64c-4107696db58a.png) ![CakeWB](https://user-images.githubusercontent.com/47809767/184556352-24c596d0-f258-4797-868e-50b82f8e7ecd.png) - **Cake** (Craft fresh baked goods in the bakery)
![Anvil](https://user-images.githubusercontent.com/47809767/184556368-195553c3-38df-47de-86b2-e94c84246480.png) ![AnviWBl](https://user-images.githubusercontent.com/47809767/184556370-8b52a0b7-bfa0-4074-bacb-27868632d36c.png) - **Anvil** (Visit the blacksmith and craft instruments)



### **Аchievements that do have a level of rarity:**
![Mint](https://user-images.githubusercontent.com/47809767/183981257-452dfe4e-e69a-4d15-9a36-24f5038c918d.gif) - **Mint**
![Mint wood](https://user-images.githubusercontent.com/47809767/183981299-a79240d3-d139-493c-af36-43ebd876c347.png) ![Mint stone](https://user-images.githubusercontent.com/47809767/183981303-47282970-65d3-4cd2-a3a5-1ff36d40a27c.png) ![Mint iron](https://user-images.githubusercontent.com/47809767/183981374-ad56b8f7-b6b5-4655-94bb-091bf4026ea2.png) ![Mint gold](https://user-images.githubusercontent.com/47809767/183981387-d9e85383-6160-48e4-942c-7a0a45cab2c1.png) ![Mint dimond](https://user-images.githubusercontent.com/47809767/183982356-9d65508c-8137-4100-9399-27a51b7d2d54.png) ![Mint woodWB](https://user-images.githubusercontent.com/47809767/183982090-f540bb49-b2e5-4f68-829e-24c02a7e2423.png) ![Mint stoneWB](https://user-images.githubusercontent.com/47809767/183982109-4fa77015-6112-4db0-9703-f1172321926d.png) ![Mint ironWB](https://user-images.githubusercontent.com/47809767/183982126-aedffb8e-5a20-4ba3-881b-7438d4bb6dd6.png) ![Mint goldWB](https://user-images.githubusercontent.com/47809767/183982149-9183ee28-c175-4b1a-813f-e07abced1829.png) ![Mint dimontWB](https://user-images.githubusercontent.com/47809767/183982168-1b27dfc0-cd99-4ebf-84fd-b22e5297c2a8.png) - **Rarity:** wood\stone\iron\gold\dimont coins (earn 10\100\1000\5000\10000 sfl)
![Cut down the tree](https://user-images.githubusercontent.com/47809767/184716823-6f29b989-1c71-4d29-bfca-6b04503f0998.gif) - **Cut down the tree**
![Cut down the tree1](https://user-images.githubusercontent.com/47809767/184716870-7a00ab63-e705-4353-a327-343fa46a5e4e.png) ![Cut down the tree2](https://user-images.githubusercontent.com/47809767/184716877-3837fedb-582b-41c9-a35e-c68dea50510c.png) ![Cut down the tree3](https://user-images.githubusercontent.com/47809767/184716885-0a8b804b-9b4a-4a65-8def-a19420cb117f.png) ![Cut down the tree4](https://user-images.githubusercontent.com/47809767/184716902-f87f4765-db21-405f-b916-556a35c9ebf1.png) ![Cut down the tree5](https://user-images.githubusercontent.com/47809767/184716922-920c200f-bb58-4c50-9a6a-6a9d8066b17d.png) ![Cut down the treeWB1](https://user-images.githubusercontent.com/47809767/184716939-ac883381-9a54-4234-8640-18e43133cac1.png) ![Cut down the treeWB2](https://user-images.githubusercontent.com/47809767/184716959-5e6bf3a5-c8b3-482e-84a3-c1cd32739e2c.png) ![Cut down the treeWB3](https://user-images.githubusercontent.com/47809767/184716978-aa54c2a7-3b3f-43d9-bc16-2389c67cdcbf.png) ![Cut down the treeWB4](https://user-images.githubusercontent.com/47809767/184716984-b87f9527-5551-44fc-b1f2-a7b37314f0a4.png) ![Cut down the treeWB5](https://user-images.githubusercontent.com/47809767/184716997-17e2d43d-c91c-4393-8f77-f1f7c7ccaf07.png) - cut down 5\25\100\500\1000 trees



> Discord TaRaSiSt#7782
> Farm id 28877
> ^_^

**Reply by Life-Is-Great** [reply]
🔥 I will definitely contribute m8 =D
Few ideas in my pocket that I'll post soon 💪

**Reply by adamhannigan** [reply]
Some great ideas 👌 Don't forget to upload the version without the background

**Reply by TaRaSiSt** [reply]
> Some great ideas 👌 Don't forget to upload the version without the background

Yes of course

**Comment by jpeyrol** [comment]
Hi, Adam

We actually tried to do something with achievements a while back. Here is the PR #817. Hope it can maybe help.

I think it would be great for everyone to have an incentive on those achivements. 

I would get a achievement for each ressource in the game and for how much you get of this ressource (for exemple: if you harvest 100 sunflower, you pass the level one of the achievement. And you would have multiple level this specific achivement.)

I would love to see like a big reward for those who complete all achivements.

**Reply by adamhannigan** [reply]
That's a good point. What sort of reward do you have in mind for this?

**Reply by jpeyrol** [reply]
> That's a good point. What sort of reward do you have in mind for this?

I would see two branches of achivements: one from the main tree and some other extras

-From the main tree:

Achievements ressources related will all be attached to this tree.

I would see a small reward when going from achieving the level 1 of achivement and so on.
For exemple, as done it the PR, when collecting a certain amout you get a small reward (for 1000 sunflower you harvested, you get 10 sunflower). Even if it's a small amount if will give some sort of incentive of the user to continue working on those achivements.

But for the last level of the achievement, this is where I give the player the badge of this specific achivements. So I would see at least one achivements badge for each ressources we have in the game currently.
Also, when finishing an achivement, you could get a title, like "Wheat Master" that could be added in your profile.

And when a user collect all the achivements badges, there will be a reward, like a decorative NFT, so other that he spent a lot of time on those achivements.

-Extra achivements:

I would see those as event related achivements, like "Collect All 10 Cakes". For the reward of those, everything depends on the difficulty.

**Comment by kakusio** [comment]
Achievements 

1- Harvest crops: 10, 100,, 1000 ... 1,00,000 ->>> 10^n
1% faster crops, up to 6%

2- Tax payer: 20, 40,, 80 ... 10,240 ->>  20 * 2^n
1% less tax, up to 10%.
so instead of paying 3% you will pay 2.7% on the max level.

3- Let it burn, burn 1k sunflowers, 1k potatoes, 1k pumpking ..... 1k wheat ->> 1k * crop
1% higher sell value, up to 10%

4- NFT holder, 1 point for each unique NFT. ->> n 
1 max stamina per day for each NFT ( after eating is implemented)

**Comment by sfwhat** [comment]
**_Some 12x12 pixel artwork ideas._**

Champion/Lucky Farmer/War veteran/Yin yang/Random medal

![ach](https://user-images.githubusercontent.com/95937808/183783536-4b4b191a-c24a-48ae-9252-a11d893ebfb6.png)
![1](https://user-images.githubusercontent.com/95937808/183783548-d58cd51c-7efb-46de-bb14-f370fd53736b.png)![2](https://user-images.githubusercontent.com/95937808/183783550-9746e0ef-6934-44bf-8b23-040e5c4c2888.png)![3](https://user-images.githubusercontent.com/95937808/183783552-64c38d81-5b39-466e-93b3-d0256f4da99c.png)![4](https://user-images.githubusercontent.com/95937808/183783555-d43a399b-22fb-46cd-93c1-209d9a140702.png)![5](https://user-images.githubusercontent.com/95937808/183783557-1e1e926d-aa00-4eaf-8a6f-94852afd8fe7.png)![6](https://user-images.githubusercontent.com/95937808/183783559-8bbecab1-3593-436b-b1e2-6a0ac567d29a.png)

Just wanted to give it a shot artistically, definitely will contribute more when ideas settle.

---
SfWhat 
Farm #6127

**Comment by Life-Is-Great** [comment]
### After spending xxx waiting time on
* Planted seeds (**Seed Dominator**)
* Mined resurces (**Miner Dominator**)
* Feeded animals (**Animal Dominator**)

### We get a dominator badge/Achievement.

![Dominators(Frame)](https://user-images.githubusercontent.com/108883042/183878716-6a12f4b6-ccf0-4744-8aa7-b2be4ecb73a8.gif)


![Seed_Dominator(Sized)](https://user-images.githubusercontent.com/108883042/183879634-75d34e6a-2ca5-4c29-bfa2-8ad3277510c9.png)  ![Miner_Dominator(Sized)](https://user-images.githubusercontent.com/108883042/183880081-6d1bb578-5b47-4cd1-afae-7f7639eee2b5.png) ![Animal_Dominator(Sized)](https://user-images.githubusercontent.com/108883042/183880109-a37379ab-7d91-4034-977a-f7ab12ca8434.png)

![frame1](https://user-images.githubusercontent.com/108883042/183880816-c3d93ad9-b4aa-48a5-b522-aeb56b447a2e.png) ![Seed_Dominator](https://user-images.githubusercontent.com/108883042/183880364-8f982cb8-5f62-469b-9a1b-0df290428ead.png)  ![Miner_Dominator](https://user-images.githubusercontent.com/108883042/183880376-8900f64c-55e0-4292-bf08-984b3aca75a7.png)  ![Farmer_Dominator](https://user-images.githubusercontent.com/108883042/183880401-14c7cd99-3e6d-40bd-b0ed-cf15758309b3.png)

**#######################################################**

### After using xxx (For crafting expanding or minting)
* Plants (**Seed Hero**)
* Mined resurces (**Miner Hero**)
* Animal resources (**Animal Hero**)

### We get a hero badge/Achievement.
![Heroes(Frame)](https://user-images.githubusercontent.com/108883042/183882709-1157b049-74d3-4547-8f58-4e4d39136dc3.gif)

![Seed_Hero(Sized)1](https://user-images.githubusercontent.com/108883042/183884254-81d94112-d491-436d-9640-e49df24f5cc2.png) ![Mining_Hero(Sized)](https://user-images.githubusercontent.com/108883042/183884388-8522655f-0aa9-4780-b68c-11a3e3cf7159.png) ![Animal_Hero(Sized)](https://user-images.githubusercontent.com/108883042/183884409-15bee5b8-fafa-48ff-8a03-4b75704e6bde.png)

![ResourceBurner_frame](https://user-images.githubusercontent.com/108883042/183883411-7153876c-9347-46df-a0a9-3970e69f2f74.png) ![ResourceHero_planting](https://user-images.githubusercontent.com/108883042/183883701-46efb5fe-3fb2-4185-9476-f6d090a945ba.png) ![ResourceHero_mining](https://user-images.githubusercontent.com/108883042/183883729-8ca48883-deab-47d0-8ae7-8ab66509f2c3.png) ![ResourceHero_farming](https://user-images.githubusercontent.com/108883042/183883740-399c2d36-6a48-467c-95ef-4c7d69c7996d.png) 


#######################################################
#######################################################

### After burning xxx of resources you get this badge/Achievement:
**(Resource burner)**
![ResourceBurner_animation(Frame)](https://user-images.githubusercontent.com/108883042/183887068-49876421-4e9e-49d0-a9fa-1e668604d7b9.gif)
![frame1](https://user-images.githubusercontent.com/108883042/183887184-29e02e05-ae4f-4e4f-be9f-eae9c12a7fa3.png) ![ResourceBurner_animation](https://user-images.githubusercontent.com/108883042/183887107-01f27c53-f21d-468b-9ebe-ba81af1b6496.gif) ![ResourceBurner_png](https://user-images.githubusercontent.com/108883042/183887081-60e87e88-9aca-415f-8e91-e7a1e1dab0ab.png)




Discord:
GodIsGreat#1718

Farm nr:
146323

**Comment by HadesCrypto** [comment]
how fast can you find the shovel 
how fast can you plant the 23 seeds 
how fast can you cut the 5 trees

the game should have a timer developed obviously to track these

**Comment by eodestroyer** [comment]
What if some of the achievements listed were also temporary buffs that could be turned on for a set period, then recharge when you hit the achievement again.  Like... after 1k wood chopped achievement(call it "Lumber Collector") -- That badge would give you 12 hours of no axes required.  Then, 10k wood, would be the tier 2 version.. ("Lumber Maverick") -- Maybe 2 days no axe req.  Then Tier 3 would be 100k("Lumber Pro") -- 1 week no axe req. 
--- All numbers can be adjusted for scale.  There could also be a burn tax on all collected lumber while using the buff.  Maybe a 10% or so that drops in the burn pile---

**Comment by Tranquilise** [comment]
As a player, I'd like when I earn rare achievements that I would also receive a POAP as a bookmark. Thanks

**Reply by adamhannigan** [reply]
Great idea 👍

**Comment by NanndoSnowzera** [comment]
**the arts I made are just an example of what can be done. it's not a definitive thing to get into, the main thing to consider is the genius idea I had, to value animal specialists. follow the examples.**

**Animal feed specialist (number of times all animals were fed).**

**Animal Feed Specialist 1** _(fed the animals 300 times)._
**Buff:** reduces the Wheat/Radish/Corn/Any other food  required to feed a Animals by 3%.

**Animal Feed Specialist 2** _(fed the animals 600 times)._
**Buff:** reduces the Wheat/Radish/Corn/Any other food  required to feed a Animals by 6%.

**Animal Feed Specialist 3** _(fed the animals 900 times)._
**Buff:** reduces the Wheat/Radish/Corn/Any other food  required to feed a Animals by 9%.

**Animal Feed Specialist 4** _(fed the animals 1300 times)._
**Buff:** reduces the Wheat/Radish/Corn/Any other food  required to feed a Animals by 15%.

**Animal Feed Specialist 5** _(fed the animals 2000 times)._
**Buff:** reduces the Wheat/Radish/Corn/Any other food  required to feed a Animals by 25%.

I made an example by putting Wheat, Radish and Corn
but can be other foods instead.
this will affect all types of animal feed.
I believe that the food system
has spaces to receive buffs in those who want to
specialize in animals, there are people who are 
buying eggs for below wheat prices today, 
this can help encourage animal breeders 
in the long run.

the arts are just examples of what can be done.
I give full freedom if any pixel art expert wants
to develop something for this idea, thank you.

**Examples:** 

![GERAL](https://user-images.githubusercontent.com/105903391/186418850-5fad8491-e600-4808-b575-ede4bf1064b7.png)


**Discord:**
Nanndo ❄ - [Snow House] - ❄#4414

**Reply by NanndoSnowzera** [reply]
**I also think the idea of ​​a person specializing in a culture is cool, for example:
we can have wheat specialist, carrot specialist, pumpkin specialist, radish specialist. in addition to running out of seed stock and the person planting other seeds before sync on chain, maybe some people want to pay more sync fees to reach the specialty of the crop they want faster.
it would be cool. and it makes perfect sense since we have large farms around the world that produce a specific crop. because they are experts in that particular culture.**
**this makes perfect sense for our playstyle, it would be a mistake to do global specials and not do cult-specific specialties,  and possibly this would increase daily sync fee collections by an average of 10 ~ 20% to help the project in the long run. the person simply has a choice, he can continue planting and synchronizing little or if he wants to become an expert of a crop he will do more synchronizations.**
